tests/org.eclipse.n4js.xpect.tests/model/validation/controlflow/DeadCode_functions.n4js.xt

```diff
@@ -27,7 +27,7 @@ function f0() {
 }
 
 // Dead Code after break
-// XPECT deadCode --> dead code
+// XPECT noerrors --> dead code
```

Comment:
I don't get it. What was the purpose of the old "deadCode" Xpect method?

WHat I find confusing is that `// XPECT noerrors --> dead code` now appears in many places where, a few lines further down, there is actually a dead code warning expected.

If the "XPECT noerrors" is actually intended, replace the "dead code" / "no dead code" string by a string actually explaining what error should not occur at this line. Otherwise, just delete the "XPECT noerrors" altogether (as you did with most cases of the old "XPECT returnOrThrow").

Reply:
True. The reason I replaced it here with a "noerrors" is to ensure that there is no "Missing return or throw" error. I replaced the misleading comment "dead code" accordingly.